stable/hazelcast/values.yaml

```diff
@@ -47,6 +47,7 @@ hazelcast:
               <discovery-strategy enabled="true" class="com.hazelcast.kubernetes.HazelcastKubernetesDiscoveryStrategy">
                 <properties>
                   <property name="service-name">${serviceName}</property>
+                  <property name="namespace">${namespace}</property>
```

Comment:
is this a backwards compatible change - or should the major version of the chart be changed instead?

Reply:
It's a bug fix. So I think changing the "patch" version is fine.

In the sense of the backwards-compatibility, yes, it is backward compatible.